stack.yaml

```diff
@@ -31,15 +31,17 @@ packages:
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
 extra-deps:
-- git: https://github.com/TorXakis/text-via-sockets.git
-  commit: e3228cd0407ec0d7991a544e154ea2def184fcae
+- git: https://github.com/gijsvcuyck/text-via-sockets.git
```

Comment:
Is there a reason not to merge this fork back first? Then you wouldn't need to change this reference.

Reply:
> Why not merge this fork back first? Then you won't need to change this reference.

I just made both pull requests at the same time, since I figured it might take a while before anything got merged. The other pull request is [here](https://github.com/TorXakis/text-via-sockets/pull/5).
It would indeed be better to merge the the test-via-sockets changes first, but it also would not be that hard to change the reference again later.